flask/templates/NBP.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>ZAKUP WALUT</title>
+</head>
+<body>
+<h1>Ile kosztuje waluta na dzień {{current_date}} po kursie NBP</h1>
+ <form method="POST" action="/NBP/">
+  <div class="container">
+      <lebel for="currency"><b>Wybierz walutę:</b></lebel>
+      <select name="currency" id="currency">
+          {% for type in currency_type %}
```

Comment:
Nie powinno nadpisywać się funkcji wbudowanych jak 'type', 
Może zamiast `currency_type` to `currency_types` i w pętli `for currency_type in currency_types`?